content/en/notebooks/advanced_analysis/_index.md

```diff
@@ -0,0 +1,122 @@
+3. Enter your query. Reserved attributes from the filtered logs are automatically added as columns.
+
+**From the [Log Explorer][1]**:
+
+1. Enter your query in the Log Explorer.  
+2. Click **Analyze in Notebooks**.  
+3. Check the **Use as a computational data source** box and select the notebook you want to use.   
+4. A data source cell is added to the selected notebook with the same query you entered in the Log Explorer. By default, the columns shown in the Log Explorer are included in the data source cell.
+
+## Configuring a data source cell
+
+After adding a data source cell to a notebook, you can continue modifying it to structure the data to suit your analysis needs. 
+
+### Change the time frame for the data
+
+By default, data source cells created from Notebooks use the global time frame. Data source cells created from the Log Explorer use a local time fixed to the time frame at the time of export.
+
+You can switch any data source cell between a local or global time frame using the toggle button in the top right corner of the cell. 
+
+### Filter the data source
+
+Regardless of how you create the data source cell, you can modify the query using the search bar. Any changes to the query automatically re-run the data source cell and any downstream cells, updating the preview of the data. 
+
+### Add or modify a column
+
+You can add or modify columns in your data source cell. There are two ways to adjust the columns:
+
+- In the preview section, click **columns** to search through available attributes for your data source.   
+- In the preview, click on a row to open the detail side panel. Click the attribute you want to add as a column and from the pop up option, select Add "@your_column" to your "@your_datasource" dataset.
+
+## Transforming and analyzing data
+
+You can add various cell types to enhance your analysis capabilities. These cells allow you to include additional data sources such as reference tables, use DDSQL to join data, and transform, correlate, and visualize your data effectively. One of the key benefits of this approach is that cells that depend on other cells are automatically updated whenever a dependency changes, ensuring your analysis always reflects the most current data.
+
+### Transformation cell
+
+Add a transformation cell to filter, group, or join, or extract data defined in a data source cell.
+
+1. Type `/transformation` and press <kbd>Enter</kbd> or click on the transform dataset tile at the bottom of the page.   
+2. Select the data source you want to transform in the source dataset dropdown menu.
+
+After adding the transformation cell, you can add any number of transformation operations inside the cell. Choose an operation from the list of supported transformations:
+| Operation | Description |
+|-----------|-------------|
+| Parse | Enter [grok syntax][2] to extract data into a separate column. In the "from" dropdown menu, select the column the data is getting extracted from. |
```

Comment:
For all these operations, the log workspaces documentation shows specific examples, can we do that too?

Reply:
@joparisot, I added calculated fields. Thanks!